evm

```diff
@@ -626,4 +626,13 @@ function _evm_complete() {
   return 0
 }
 
-complete -F _evm_complete evm
+if [ -n "$(command -v complete)" ]; then
+{
+  complete -F _evm_complete evm
+}
+fi
+if [ -n "$(command -v compdef)" ]; then
```

Comment:
Do you mind using an `elif` here instead of falling through `compdef` ? Also, do you mind dropping the brackets `{}`, I am not sure you really need those.

You probably should also send the output of the command to `/dev/null` instead of returning from the subshell and use a simple if check, like:

```
# Setup auto complete giving priority to complete command and falling back to compdef if it exists 
if command -v complete >/dev/null; then
  complete -F _evm_complete evm
elif command -v compdef >/dev/null; then
  compdef _evm_complete evm
fi
```

Reply:
@robisonsantos Done